rfcs/0093-rfc-categories.md

```diff
@@ -0,0 +1,152 @@
+venues to a broder RFC author base.
+
+## Add structure to an increasingly used RFC process
+
+As the RFC process will be more widely used by a growing community, it becomes necesary to
+structure and differentiate the process further to remain as efficient and accessible as
+possible. Adding a category metadata and differentiate the templates will help to better
+accomodate the multiple aspects that might require a "Request For Comment" from the broader
+community. It is expected that we identify more useful categories and nuances to the process
+as its usage increases.
+
+# Detailed design
+[design]: #detailed-design
+
+Every RFC that is eligible for the RFC process is classified by its author into the
+_informational_, _process_ or _feature_ category. For each category a different template
+is made available. How those categories are defined in every detail can remain
+subjective, but the following should give a sufficient idea:
+
+- Informational RFCs
+  - Start a talk, meetup, or social networking account that will be expected to officially “represent nix”
+  - Document design issues, deciding to never implement a feature
+  - Proposing an experiment
+  - Recording a proof-generated insight 
+- Process RFCs
+  - Change the RFC process, the organization of the issue tracker or the support forum
+  - Changes to community workflows or other community infrastructure
+  - Amend the Code of Conduct        
+- Feature RFCs
+  - Anything that is currently covered by the RFC process and does not better fit into
+    any of the other two categories.
+
+Authors are advised to choose the most prevalent category for their classification. For
+example, if an informational RFC requires some changes to the community infrastructure,
+but still mainly proposes an experiment, it would go into the "informationl" category.
+
+This RFC is accompanied by commits that implement it. Please refer to them for the detailed
+design.
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+The relevant examples and interactions are exposed in the motivation section.
+The detailed design does not require further exemplification, since it is intended to be
+an authors personal best judgment that drives the categorization, not a specific set of criteria.
+I don't think its feasible to develop such fixed set of criteria herein in a meaningful way.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+No substantial drawback comes to my mind, that would not be an alibi for this section.
+
+# Alternatives
+[alternatives]: #alternatives
+
+No alternative comes to my mind, that would not be an alibi for this section.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+At the point of initiating this RFC, the detailed changes to the template are not yet known.
+They will be complemented at any suitable point before reaching FCP.
+
+# Future work
+[future]: #future-work
```

Comment:
Future work is probably also to reevaluate the categories.

1. Are there categories that are unused?
2. Are there categories which are discussed in a very similar manor and are suitable for merging.
3. Are there categories which have different types of discussion evolving which may be better handled by splitting the category into multiple.

This may include collapsing back into one category if this RFC doesn't seem to help the discussions.